recipes/seeq/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "seeq" %}
+{% set version = "0.0.117" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ab83336f21191c9c66026315de2b2889f3f86cfc44e1475df7b62895be9c1530
+
+build:
+  number: 0
+  skip: True  # [py<36]
```

Looks like this can be noarch:
```suggestion
  noarch: python
```